### ISSUE TITLE
If a user has edit access, we should record they have read access too

### ISSUE THREAD

**Issue by jcoyne** [projecthydra/hydra-head#341]
This will allow Hydra ACLs to align with Fedora ACLs.

**Comment by atz** [comment]
Yes, very much agree.  The abstract case is "should explicitly index lesser permissions implied by greater ones".  This would simplify downstream querying greatly.
